okteto.yml

```diff
@@ -0,0 +1,9 @@
+name: hello-world
+image: mcr.microsoft.com/dotnet/core/sdk
+environment:
+- ASPNETCORE_ENVIRONMENT=Development
+command:
+- bash
+workdir: /usr/src/app
```

Comment:
can we remove `workdir` and use the default?

Reply:
That's the workdir of the dev image, I want to keep it